docs/reference/projectyml.rst

```diff
@@ -54,6 +54,34 @@ The code snippet below provides an example of a complete ``project.yml`` file:
       index_urls:
         - https://pypi.org/simple/
 
+
+.. _specify_location_pip:
+
+Specify locations from where V2 modules will be installed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By setting ``use_config_file`` option to ``True``, the project will use the global pip config file when installing V2 modules.
+To specify the url of a pip repository, add the following to the pip config file:
+
+.. code-block:: text
+
+  [global]
+  timeout = 60
+  index-url = <url of the repository >
+
+.. note::
+
+    You can also create a new pip config file ``pip.conf``, add the content there and then set the ``PIP_CONFIG_FILE`` environment variable to the path of the newly created ``pip.conf`` file (See: :ref:`env_vars`).
+
+Another option is to use the  ``index_urls`` option. There you can directly add the urls of repositories you are interested in.
```

Comment:
We should make it clear that the `index_urls` config option is a config option of the project.yml file. Maybe also refer to it as `pip.index_urls`, because index_urls is a sub-config option of the pip section.